pyvrp/educate/tests/test_Exchange.py

```diff
@@ -206,3 +211,41 @@ def test_relocate_after_depot_should_work():
     individual = Individual(data, [[1, 2, 3], [4]])
     expected = Individual(data, [[1, 2], [3], [4]])
     assert_equal(ls.search(individual, cost_evaluator), expected)
+
+
+def test_reoptimize_changed_objective_timewarp_OkSmall():
```

Comment:
Refactor: this test relates to LS, not Exchange. I think it belongs in `test_LocalSearch.py` instead.